### DISCUSSION TITLE
Unsubscribing from DelegateCommand.ObservesProperty

### DISCUSSION THREAD

**Discussion by james1301** [PrismLibrary/Prism#2578] [WPF] [answered]
Is there a need to unsubscribe from Observes Property in anyway? Looking at the code it hooks up the PropertyChanged event directly to the object, but I can't see when that would be unsubscribed. I do see an unsubscribe private method, but don't quite follow when that would occur.

**Comment by brianlagunas** [comment, marked as answer]
Nope.
